tools/gen_rotating_star_init/make_bin.cpp

```diff
@@ -193,7 +193,7 @@ int main()
         printf("%e %e %e %e %e\n", omega, virial, T, W, K);
 
         scf_iter++;
-        if (scf_iter > 20)
+        if (scf_iter > 50)
```

Comment:
This seems albeit non-relevant for this PR, but I approve it anyway.
@dmarce1, can you explain why this change is necessary?